### ISSUE TITLE
Training LCD numbers

### ISSUE THREAD

**Issue by albundy0815** [jomjol/AI-on-the-edge-device#755]
Hi, would you please try to implement it into the training? I hope that the numbers are ok, because I've some issues with the lightning (indirect LEDs
[AI-Stromzaehler.zip](https://github.com/jomjol/AI-on-the-edge-device/files/8417886/AI-Stromzaehler.zip)
)

**Comment by jomjol** [comment]
Sorry, I cannot use this images for the training. They do not have the correct ROI. Can you adjust it like the following: https://github.com/jomjol/AI-on-the-edge-device/wiki/ROI-Configuration

The 20% free border is needed, as it is necessary for the automatic shift of the augmentation.

Additionally the contrast is really not good. It makes no sense to train them like this.
![8_10_20220405-045641](https://user-images.githubusercontent.com/30766535/161799379-35e2654b-a5b9-4d3b-a25b-3f7af14ef1cc.jpg)

**Comment by haverland** [comment]
I think its a good idea to train images are not so good. It makes the model more failure tolerant. The Batchnormalization should normalize the dark image.

**Comment by jomjol** [comment]
The problem is not the darkness, but the missing contrast from the LCD to the background. I will not incorporate this images to the general training, but you can make a dedicated training set for your usage.
There is a github to do this also available public:

- automated version: https://github.com/jomjol/neural-network-autotrain-digital-counter
- manual version: https://github.com/jomjol/neural-network-digital-counter-readout

**Comment by haverland** [comment]
@albundy0815 you could try a this model https://github.com/haverland/neural-network-autotrain-digital-counter/blob/main/script_manual/kndq.tflite?raw=true

I trained it with your images. All LCDs should be recognized.
This are the images are not correct predicted:
<img width="1072" alt="image" src="https://user-images.githubusercontent.com/412645/161844343-2da60c17-c583-4fe7-b1c3-7b93d504ffe8.png">


@jomjol: yes on a small CNN it's not easy. It could be more sharpen (focus of the camera?). But a good model could be better recognize it than a human. 11 classes are not much.

**Comment by albundy0815** [comment]
@haverland thanks for the tflite file. I'll check it soon
@jomjol I try to get a better image quality by increasing the distance from power meter -> ESP. I guess, that the quality will getting better. Thanks a lot for the whole project!

**Comment by jomjol** [comment]
No acitivies anymore --> closing
